packages/ui/src/theme/tokens/components/switchField.ts

```diff
@@ -92,6 +93,9 @@ export const switchfield: SwitchFieldTokens = {
   large: {
     fontSize: { value: '{fontSizes.large.value}' },
   },
+  medium: {
```

Comment:
I believe the pattern we use with the default (medium) size is to add the tokens directly onto the component instead of using a medium prefix.

Reply:
Good catch, you are right